src/_Problems_/merge-two-sorted-arrays/index.js

```diff
@@ -38,3 +38,8 @@ function mergeTwoSortedArrays(arr1, arr2) {
 function mergeTwoSortedArrays2(arr1, arr2) {
```

Comment:
this is a algo question, so you are expected not to use sort function, please write sort function or whatever logic you think is the optimal

Reply:
I got your concern but I am putting forward all the possible ways!